accounts-db/src/accounts_index/stats.rs

```diff
@@ -62,6 +62,15 @@ pub struct Stats {
     bins: u64,
     pub flush_should_evict_us: AtomicU64,
     pub flush_read_lock_us: AtomicU64,
+    // Max flush stats for 10s reporting interval
+    pub flush_max_total_entries_before: AtomicU64,
+    pub flush_max_num_flushed: AtomicU64,
+    pub flush_max_map_len_before: AtomicU64,
+    pub flush_max_map_capacity_before: AtomicU64,
+    pub flush_max_map_len_after: AtomicU64,
+    pub flush_max_map_capacity_after: AtomicU64,
+    pub flush_max_evicted: AtomicU64,
+    pub flush_max_total_entries_after: AtomicU64,
```

Comment:
I'm not sure I understand why 'max' is useful here. We can get the max via Influx already.

Reply:
I see. I'll take another look, thanks!